src/Kernel/Aggregates/Configuration.php

```diff
@@ -666,7 +666,7 @@ public function getBoletoDueDays()
      */
     public function setBoletoDueDays($boletoDueDays)
     {
-        $this->boletoDueDays = $boletoDueDays;
+        $this->boletoDueDays = (int) $boletoDueDays;
```

Comment:
Não há possibilidade de declarar esta variável como int para evitar o cast? Há possibilidade de ocorrer erro nesse cast? Se sim, seria interessante tratar isso, não?

Reply:
como essa info vem de um formulario do Magento, ela vem como string, vou adicionar o tratamento que fazemos no attributo amount, onde verificamos se é numerico e se nao for retornamos um erro. De qualquer forma terei que fazer este cast aqui.